docs/src/linux.md

```diff
@@ -29,6 +29,7 @@ There are several third-party Zed packages for various Linux distributions and p
 * Arch: [`zed`](https://archlinux.org/packages/extra/x86_64/zed/)
 * Arch (AUR): [`zed-git`](https://aur.archlinux.org/packages/zed-git), [`zed-preview`](https://aur.archlinux.org/packages/zed-preview),  [`zed-preview-bin`](https://aur.archlinux.org/packages/zed-preview-bin)
 * Alpine: `zed` ([aarch64](https://pkgs.alpinelinux.org/package/edge/testing/aarch64/zed)) ([x86_64](https://pkgs.alpinelinux.org/package/edge/testing/x86_64/zed))
+* AOSC OS: [`zed`](https://packages.aosc.io/packages/zed)
```

Comment:
Can we add this below `ALT Linux (Sisyphus)`?

Reply:
Done